src/setup.py

```diff
@@ -1,48 +1,71 @@
 import os
-from setuptools import setup, find_packages
+import sys
+
+from setuptools import find_packages, setup
+from setuptools.command.test import test as TestCommand
 
 base_path = os.path.abspath(os.path.dirname(__file__))
 
 requirements = []
-with open(os.path.join(os.path.dirname(base_path), 'requirements.txt')) as f:
+with open(os.path.join(os.path.dirname(base_path), "requirements.txt")) as f:
     requirements = f.read().splitlines()
 
-readme = ''
-with open(os.path.join(os.path.dirname(base_path), 'README.md')) as f:
+tests_requirements = []
+with open(os.path.join(os.path.dirname(base_path), "requirements_tests.txt")) as f:
+    tests_requirements = f.read().splitlines()
+
+readme = ""
+with open(os.path.join(os.path.dirname(base_path), "README.md")) as f:
     readme = f.read()
 
+
+class PyTestCommand(TestCommand):
```

Comment:
What is this class used for? Also has it been tested to confirm it works?

Reply:
This class is launched when you execute `python setup.py test`, it will run the tests for you.

Reply:
Ah alright, does it affect the pypi package when it's updated?

Reply:
I don't think so, but I'll remove it to make it safe